src/github.com/getlantern/flashlight/genconfig/cloud.yaml.tmpl

```diff
@@ -1,6 +1,7 @@
 # cloud.yaml contains the default configuration that's made available on the
 # internet.
 client:
+  firetweetversion: "0.0.6"
```

Comment:
Putting the version file here: https://raw.githubusercontent.com/firetweet/downloads/master/version.txt
We can just update that file whenever we deploy the release APK as you said